scripts/dcp1-version/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM quay.io/ebi-ait/ingest-base-images:python_3.7-slim
+
+COPY ./populate_dcp_version.py populate_dcp_version.py
+COPY ./requirements.txt requirements.txt
+COPY ./dcp1-metadata.txt.tar.gz dcp1-metadata.txt.tar.gz
+RUN tar -xzvf dcp1-metadata.txt.tar.gz
+
+ENV DCP1_GS_FILES_LIST=dcp1-metadata.txt
+
+RUN pip install -r requirements.txt
```

Comment:
Not really neccesary here but you could use multi stage builds to speed things up a bit